pkg/helpers/describe/projectstatus.go

```diff
@@ -265,7 +265,7 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
 		if allNamespaces {
 			fmt.Fprintf(out, describeAllProjectsOnServer(f, d.Server))
 		} else {
-			fmt.Fprintf(out, describeProjectAndServer(f, project, d.Server))
```

Comment:
`describeProjectAndServer` is not used by anyone. Can we also remove it as well?.

Reply:
> `describeProjectAndServer` is not used by anyone. Can we also remove it as well?.

Tidied up along with the project client.